.github/ci_tests/posix.yaml

```diff
@@ -11,7 +11,7 @@ global:
     host: "broker-sda-mq"
     exchange: "lega"
     port: 5671
-    verifyHostname: false
+    verifyPeer: false
```

Comment:
It seems to work without `verifyPeer` now that we've managed to fix the other issues (https://github.com/neicnordic/sda-helm/runs/1031804731?check_suite_focus=true).

Reply:
That's good to know. I am not sure how feasible it would be to create different integration secenarios using different settings...